scripts/linters/ts_ignore_exceptions.json

```diff
@@ -12,5 +12,9 @@
   ],
   "core/templates/tests/unit-test-utils.ts": [
     "element, [failHardModule, ng1Module.name]));"
+  ],
+  "core/templates/pages/exploration-player-page/services/content-translation-manager.service.spec.ts": [
```

Comment:
We typically try not to add anything here. @nishantwrp do you have any advice on what to do so that this can be removed? I seem to recall that we had similar karma tests elsewhere that test invalid structures...

Reply:
Hi @shavavo, you can use `@ts-expect-error` instead of `@ts-ignore`. Here is an example 

https://github.com/oppia/oppia/blob/a3f7191fe7a9db9ec32bc2bf1de044a8943fef57/extensions/interactions/EndExploration/directives/end-exploration-validation.service.spec.ts#L115-L119